### ISSUE TITLE
[Visual Requirements - Make Code - Delete Project] Elements must have sufficient color contrast (.text)

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2352]
**User Experience:**
Visual impaired user would find difficulty to see the control or element if that controls have insufficient color contrast as they won't be able to take on that controls and he/she would miss the functionality of such control or element.

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/beta
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Delete Project" option and select it.
7. Navigate through all the controls appearing under "Delete Project" dialog.
8. Verify and check the luminosity ratio of 'Delete' button control.

**Actual Result:**
Luminosity ratio is less than 4.5:1 for 'Delete' button control under 'Would you like to delete 'Untitled'?' pop-up window.

**Expected Result:**
Luminosity ratio should be more than and equal to 4.5:1 for 'Delete' button control under 'Would you like to delete 'Untitled'?' pop-up window.

**MAS Reference:**
[MAS 1.4.3 - Contrast (Minimum)] (https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6})

**Issue:** Elements must have sufficient color contrast (color-contrast: https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI)

**Target application title:** Microsoft MakeCode for micro:bit
Target application url: https://makecode.microbit.org/#editor 

**Element path:** .approve.red.labeled > .text

**Snippet:** <span class="ui text">Delete</span>

**How to fix:**
Fix any of the following:
  Element has insufficient color contrast of 2.78 (foreground color: #ffffff, background color: #ef767a, font size: 12.0pt, font weight: bold). Expected contrast ratio of 4.5:1

**Environment:**
Version 1903 (OS Build 18867.1000)
Platform: Edge, Chrome version (75.0.3770.142)

This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](https://accessibilityinsights.io/).

![MAS1 4 3_Delete Project_CCA](https://user-images.githubusercontent.com/30456910/62538120-a9bb0f00-b86f-11e9-802c-f2dc2f3a07ee.jpg)

**Comment by abchatra** [comment]
Please re-test using beta site https://makecode.microbit.org/beta . If it reproduces let me know. I will reopen the bug.

**Comment by jwunderl** [comment]
This bug is a duplicate of https://github.com/microsoft/pxt-microbit/issues/2292 - both use the same dialog with low contrast buttons

**Comment by sonimishra** [comment]
Issue still exists in https://makecode.microbit.org/beta so reactivating the issue.

**Comment by abchatra** [comment]
This should be tested under high contrast mode right?

**Comment by abchatra** [comment]
@jwunderl

**Comment by jwunderl** [comment]
Contrast issues should be tested in high contrast mode, and this behaves properly there:

![Screen Shot 2019-08-08 at 10 31 35 AM](https://user-images.githubusercontent.com/5615930/62724390-bd798900-b9c7-11e9-916a-77bb2f9cb469.png)

I still think the red should be changed as it doesn't like right to me, but wouldn't think it would count as an accessibility issue for that reason

**Comment by abchatra** [comment]
Please test  @sonimishra  mode and reopen if necessary. 

All contrast issues needs to be tested in high contrast mode. 

![image](https://user-images.githubusercontent.com/6107272/62724652-54dedc00-b9c8-11e9-8c58-c5b9b7bed306.png)
